pkgs/tools/wayland/swww/default.nix

```diff
@@ -0,0 +1,24 @@
+{ config, lib, pkgs, fetchFromGitHub, rustPlatform, pkg-config, lz4
+, libxkbcommon, ... }:
+rustPlatform.buildRustPackage rec {
+  pname = "swww";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "Horus645";
+    repo = pname;
+    rev = "v0.7.0";
+    sha256 = "ofJTL+9izMmwENaEVYQ7rjpoEFR1J2szkJxzYb09N7g=";
+  };
+  cargoSha256 = "sha256-ntulNQUQZ42CGvwnnQSMQsr/MDSNY/5/Aq2Ew9QItn8=";
+  buildInputs = [ lz4 libxkbcommon ];
+  doCheck = false; # Integration tests do not work in sandbox enviroment
+  nativeBuildInputs = [ pkg-config ];
+  meta = with lib; {
```

Comment:
You should add yourself as maintainer of the package since I expect that was your goal when you added yourself to the global maintainers list.

Reply:
Done.